tests/test_diffeq/test_odesolver.py

```diff
@@ -62,26 +62,5 @@ def test_solve(self):
         self.assertAlmostEqual(odesol.states[-1].mean[0], 1.0, places=2)
 
 
-class TestTimeStopper:
-    @pytest.fixture(autouse=True)
-    def _setup(self):
-        self.time_stops = [2.0, 3.0, 4.0, 5.0]
-        self.discrete_events = diffeq._odesolver._TimeStopper(locations=self.time_stops)
-
-        def dummy_perform_step(state, dt, steprule):
-            return state, dt
-
-        self.dummy_perform_step = dummy_perform_step
-
-    def test_adjust_dt_to_time_stops(self):
-        # Should interfere dt to 0.1 instead of 5.0, because 2 is in self.time_stops
-        dt = self.discrete_events.adjust_dt_to_time_stops(t=1.9, dt=5.0)
-        assert dt == pytest.approx(0.1)
-
-        # Should not interfere dt if there is no proximity to an event
-        dt = self.discrete_events.adjust_dt_to_time_stops(t=1.0, dt=0.00001)
-        assert dt == pytest.approx(0.00001)
-
-
```

Comment:
theses tests test the behaviour of a private method (which is suboptimal), and are redundant. Whenever I break the implementation of the time stop adjustment, other tests fail, too.